rotkehlchen/tests/db/test_db_upgrades.py

```diff
@@ -2142,6 +2142,44 @@ def test_upgrade_db_26_to_27(user_data_dir):  # pylint: disable=unused-argument
     assert db.get_version() == 27
 
 
+@pytest.mark.parametrize('use_clean_caching_directory', [True])
+def test_upgrade_db_27_to_28(user_data_dir):  # pylint: disable=unused-argument
+    """Test upgrading the DB from version 27 to version 28.
+
+    - Recreates balancer events, uniswap events, amm_swaps. Deletes balancer pools
```

Comment:
This is not what this test does? Please focus when coding, don't copy pasta so much.